services/workflows-service/src/data-migration/scripts/sync/sync.ts

```diff
@@ -281,6 +330,30 @@ export const sync = async (objectsToSync: SyncedObject[]) => {
     throw err;
   }
 };
+function mergeEnvironmentConfigs(acc: any, key: any, obj: any) {
```

Comment:
Isn't this the type? Otherwise we can use `Record<string, any>`.
```suggestion
function mergeEnvironmentConfigs(acc: Record<string, SyncedObject>, key: string, obj: SyncedObject) {
```